lib/inky/inkycommands.ex

```diff
@@ -0,0 +1,192 @@
+defmodule Inky.Commands do
+  alias Inky.InkyIO
+
+  # API
+  def pixels_to_bitstring(pixels, display, p2b) do
+    rotation = display.rotation / 90
+    width = display.width
+    height = display.height
+
+    opts = bitstring_traversal_opts(rotation, width, height)
+    do_pixels_to_bitstring(pixels, p2b, opts)
+  end
+
+  def update(pins, display, packed_height, buffer_black, buffer_accent) do
+    pins
+    |> set_analog_block_control()
+    |> set_digital_block_control()
+    |> set_gate(packed_height)
+    |> set_gate_driving_voltage()
+    |> dummy_line_period()
+    |> set_gate_line_width()
+    |> set_data_entry_mode()
+    |> power_on()
+    |> vcom_register()
+    |> set_border_color()
+    |> configure_if_yellow(display.accent)
+    |> set_luts(display.luts)
+    |> set_dimensions(display.width, packed_height)
+    |> push_pixel_data(buffer_black, buffer_accent)
+    |> display_update_sequence()
+    |> trigger_display_update()
+    |> wait_before_sleep()
+    |> deep_sleep()
+  end
+
+  def soft_reset(pins) do
+    send_command(pins, 0x12)
+  end
+
+  def await_device(pins) do
+    should_wait = InkyIO.read_busy(pins) == 1
+
+    if should_wait do
+      :timer.sleep(10)
+      await_device(pins)
+    else
+      :ok
+    end
+  end
+
+  # Bitstringifying
+
+  defp bitstring_traversal_opts(rotation, width, height) do
```

Comment:
I think we have a regression here, I'm the same offset diagonal pattern as when this was broken previously when I display stripes. So I'm guessing we are either hitting the rotation default or something like that. At the time of writing I've no time to test.